app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ForgotPasswordsController < ApplicationController
+  def create
+    user = User.find_by_email(params[:email])
```

Comment:
I would recommend to avoid dynamic finders `find_by_xxx`. They are less efficient than `find_by` and they go through `method_missing`. Rails 4 makes `find_by` top candidate to use in case of searching for a single record.